lib/repl.js

```diff
@@ -1222,7 +1224,7 @@ REPLServer.prototype.convertToContext = function(cmd) {
   }
 
   return cmd;
-};
+}, 'replServer.convertToContext will be removed after v7');
```

Comment:
Our deprecation messages typically take a form like, 'replServer.convertToContext() is deprecated', without committing to a specific version when it may be removed. The message should also indicate if there is an alternative that the developer should use.